ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh

```diff
@@ -11,10 +11,28 @@ expiration_date=$(date -d '4 hours' --iso=minutes --utc)
 REGION="${LEASED_RESOURCE}"
 case "${REGION}" in
 us-east-1)
-   ZONE_1=us-east-1b
-   ZONE_2=us-east-1c;;
+   ZONE_1=us-east-1a
+   ZONE_2=us-east-1b;;
```

Comment:
i think we have chosen b,c for us-east-1 for a reason and i don;'t think we should change it here.

Reply:
These are only used for echoing at line 17. In the original code if I am not mistaken. It is passed as a and b to the config file.

Reply:
no these are not used to just echo, but set the ZONE_1 and ZONE_2 env variables that are then used here https://github.com/openshift/release/pull/14747/files#diff-e12f1cf79ea6ed6c844819d0d23955fdaa6267109829daeeaa38237dce596532R53

Reply:
Those variable names are `ZONE1` but the variable name here is `ZONE_1`. Sorry, I am getting a bit confused.